app/root.js

```diff
@@ -6,6 +6,14 @@ const passworder = require('browser-passworder')
 
 module.exports = connect(mapStateToProps)(AppRoot)
 
+function decodeMnemonic(mnemonic) {
+  if (typeof mnemonic === 'string') {
```

Comment:
This should handle existing accounts as well as any new accounts that were created after the aforementioned PR in the extension.